test/mpi/coll/testlist.in

```diff
@@ -176,3 +176,24 @@ p_bcast 4 strict=false
 p_bcast2 8 strict=false
 
 coll_large 8
+p_allred 2 strict=false
+p_red 4 strict=false
+p_alltoall 8 strict=false
+p_allgatherg10 strict=false
```

Comment:
typo (your keyboard has issues :) )

Reply:
Yes, embarrasingly. The new keyboard firmware has some timing issues.